common/src/main/java/com/scottlogic/deg/common/util/FileUtils.java

```diff
@@ -51,11 +52,11 @@ public boolean isFileEmpty(File file) {
     }
 
     public boolean exists(Path path) {
-        return Files.exists(path);
+        return path.toFile().exists();
```

Comment:
why is this better?

Reply:
https://rules.sonarsource.com/java/tag/performance/RSPEC-3725